modules/client-configuration/pages/client-config-overview.adoc

```diff
@@ -6,3 +6,44 @@ Registering clients is the first step after installing {productname}, and most o
 {productname} is compatible with a range of client technologies: you can install traditional or Salt clients, running {sle} or another Linux operating system, with a range of hardware options.
 
 This guide discusses how to register and configure different clients, both manually and automatically.
+
+
+== Compare Traditional and Salt Clients
+
+This table lists the availability of various features for traditional and Salt clients.
+
+In this table, icon:check[role="green"] indicates that the feature is available, and icon:times[role="danger"] indicates that it is not available.
+Fields marked as icon:question[role="gray"] are under consideration, and may or may not be made available at a later date.
+
+[cols="1,1,1,1", options="header"]
+.Features of Traditional and Salt Clients
+|===
+| Feature               | Notes | Traditional               | Salt
+| Registration          |       | icon:check[role="green"]  | icon:check[role="green"]
+| Install Packages      |       | icon:check[role="green"]  | icon:check[role="green"]
+| Install Patches       |       | icon:check[role="green"]  | icon:check[role="green"]
+| Remote Commands       |       | icon:check[role="green"]  | icon:check[role="green"]
+| System Package States |       | icon:times[role="danger"] | icon:check[role="green"]
+| System Custom States  |       | icon:times[role="danger"] | icon:check[role="green"]
+| Group Custom States   |       | icon:times[role="danger"] | icon:check[role="green"]
+| Organization Custom States |  | icon:times[role="danger"] | icon:check[role="green"]
+| System Set Manager    |       | icon:check[role="green"]  | icon:check[role="green"]
+| Service Pack Migration |      | icon:check[role="green"]  | icon:check[role="green"]
+| Virtual Host Management | Bootstrap installation support | icon:check[role="green"] | icon:check[role="green"] (read-only)
+| System Redeployment | With Auto-installation | icon:check[role="green"] | icon:question[role="gray"]
+| Contact Methods | Between server and client | osad, rhnsd, ssh push | zeromq: Salt default salt-ssh
+| Red Hat Network Clients |     | icon:check[role="green"]  | icon:check[role="green"]
```

Comment:
I do not know what this means. Is this meant to be "RHEL/CentOS/Fedora/derivatives"?

Reply:
I assumed it was clients running Red Hat operating systems, which is why I added in the Ubuntu clients one beneath it.

Reply:
[Red Hat Network](https://en.wikipedia.org/wiki/Red_Hat_Network) is the project that became Spacewalk, I do not think it is an appropriate name in this context.

We should probably list OSs explicitly - technically CentOS and Fedora are not Red Hat OSs either.

AFAIK, Red Hat Enterprise Linux 6 and 7 and SUSE Linux Enterprise Expanded Support are officially supported as clients. CentOS 6 and 7, Scientific Linux, Oracle Linux and other RHEL derivatives are known to work but not officially supported. Ubuntu LTS 16.04 LTS and 18.04 LTS are officially supported. Other versions, Debian, and other Debian derivatives might also work but they are not officially supported.

Reply:
Considering we already have a supported clients matrix, I'd rather not replicate it here. Let's just remove those lines.